### ISSUE TITLE
SW-100031

### ISSUE THREAD

**Issue by aizaz-clym** [aizaz-clym/clym-test#7]
## Structural markup should be used for presentational effect

| Type | Standard | Category | Occurrences | Page | Scanned | Status |
| --- | --- | --- | --- | --- | --- | --- |
| Automatic | WCAG | Common Failures | 1 | Home | 2026-07-15 | 🔴 Failed |

### Steps

1. Check that the element's semantic meaning is exposed to assistive technology and appropriate for the content of the element.

### Criteria

| Criteria | Name | Description | Level |
| --- | --- | --- | --- |
| [1.3.1](https://www.w3.org/WAI/WCAG22/quickref#info-and-relationships) | Info and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text. | A |

### Selector

`html`

### HTML

```html
<html lang="en">
```

---

> Created automatically by [Clym](https://clym.io)

**Comment by aizaz-clym** [comment]
#Title
## subtitle

**Comment by aizaz-clym** [comment]
# Markdown syntax guide

## Headers

# This is a Heading h1
## This is a Heading h2
###### This is a Heading h6

## Emphasis

*This text will be italic*  
_This will also be italic_

**This text will be bold**  
__This will also be bold__

_You **can** combine them_

## Lists

### Unordered

* Item 1
* Item 2
* Item 2a
* Item 2b
    * Item 3a
    * Item 3b

### Ordered

1. Item 1
2. Item 2
3. Item 3
    1. Item 3a
    2. Item 3b

## Images

![This is an alt text.](/image/Markdown-mark.svg "This is a sample image.")

## Links

You may be using [Markdown Live Preview](https://markdownlivepreview.com/).

## Blockquotes

> Markdown is a lightweight markup language with plain-text-formatting syntax, created in 2004 by John Gruber with Aaron Swartz.
>
>> Markdown is often used to format readme files, for writing messages in online discussion forums, and to create rich text using a plain text editor.

## Tables

| Left columns  | Right columns |
| ------------- |:-------------:|
| left foo      | right foo     |
| left bar      | right bar     |
| left baz      | right baz     |

## Blocks of code

```
let message = 'Hello world';
alert(message);
```

## Mermaid diagrams
```mermaid
graph TD
  A[Start] --> B{Decision}
  B -->|Yes| C[Finish]
  B -->|No| D[Alternate]
```

## Inline code

This web site is using `markedjs/marked`.
